### DISCUSSION TITLE
The use of Gate

### DISCUSSION THREAD

**Discussion by liangzhixin-202169** [e3nn/e3nn#445] [Q&A]
Hello. 
I have created a tensor product(tp) that has tp.irreps_out == '112x0e+8x1o+8x2e+8x3o+8x4e+8x5o+8x6e+8x0o+8x1e+8x2o+8x3e+8x4o+8x5e+8x6o'. I want to pass the tp result to a Gate, so I set irreps_scalars='8x0e', irreps_gates='104x0e', and irreps_gated='8x1o+8x2e+8x3o+8x4e+8x5o+8x6e+8x0o+8x1e+8x2o+8x3e+8x4o+8x5e+8x6o'. 
However, when I pass the tp result to this Gate, such as gate(feature), it seems that the gate always takes  feature[:8] as '8x0o', feature[8:16] as irreps_scalars, and feature[16:120] as irreps_gates. 
I wonder is there any wrong in the parameters used to initialize the Gate object?

**Comment by mariogeiger** [comment]
What you did seems correct though... can you share some code maybe?
